content/en/docs/doc-contributor-tools/linkchecker/README.md

```diff
@@ -0,0 +1,67 @@
+# Internal link checking tool
+
+You can use [htmltest](https://github.com/wjdp/htmltest) to check for broken links in [`/content/en/`](https://git.k8s.io/website/content/en/). This is useful when refactoring sections of content, moving pages around, or renaming files or page headers.
+
+## What it does and doesn't check
+
+The link checker scans generated HTML files, not raw markdown. You can see the full configuration in [`.htmltest.yml`](https://git.k8s.io/website/.htmltest.yml).
+
+The link checker scans the following:
+
+- All content generated from Markdown in [`/content/en/docs`](https://git.k8s.io/website/content/en/docs/) directory, excluding:
+  - Generated API references
+- All internal links, excluding:
+  - Empty hashes (`<a href="#">` or `[title](#)`) and empty hrefs (`<a href="">` or `[title]()`)
+  - Internal links to images and other media files
+
+The link checker does not scan the following:
+
+- Links included in the top and side nav bars, footer links, or links in a page's `<head>` section, such as links to CSS stylesheets, scripts, and meta information
+- Top level pages and their children, for example: `/training`, `/community`, `/case-studies/adidas`
+- Blog posts
+- API Reference documentation
```

Comment:
Line 22, meaning the two HTML references (kubectl-command and API ref)?

Reply:
Once again – https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/ and its ilk. Really just reiterating here to drive the point home.